setup.py

```diff
@@ -17,7 +17,6 @@
     author_email="tech@opensafely.org",
     python_requires=">=3.7",
     install_requires=[
-        "lz4",  # implicit import for compressed feather files
```

Comment:
I worry this means that people won't be able to create measures from previous feather extracts. I think sadly we might need to leave it in for a bit until we've confirmed that no one needs to do this.

Reply:
Curse you lz4. This day you survive again, but I will kill you one day.